source/distributed/tria_base.cc

```diff
@@ -1442,12 +1443,13 @@ namespace parallel
       // this task.
       if (myrank == 0)
         {
-          ierr = MPI_File_write_at(fh,
-                                   0,
-                                   sizes_fixed_cumulative.data(),
-                                   sizes_fixed_cumulative.size(),
-                                   MPI_UNSIGNED,
-                                   MPI_STATUS_IGNORE);
+          ierr = Utilities::MPI::LargeCount::MPI_File_write_at_c(
```

Comment:
This is not strictly necessary, as the header will not exceed 2GB, but this is what @bangerth suggested in his review of the other PR. So let's do the change here as well.

Reply:
Appreciated!